.meta/transforms/regex_parser.toml

```diff
@@ -39,4 +39,23 @@ description = """\
 The Regular Expression to apply. Do not include the leading or trailing `/`.\
 """
 
+[transforms.regex_parser.options.overwrite_target]
```

Comment:
I don't think we need this option. Other transforms simply overwrite existing values, which I think is a fine default.

Reply:
As indicated to @Hoverbear, this mirrors the `json_parser` transform. Since it is the only other parser transform that includes the `target_field` capability, I thought it made sense to copy its behavior as well.

Reply:
Ah, I wasn't aware we had an `overwrite_target` option. Ignore my comment then.